### ISSUE TITLE
Not working in webstorm

### ISSUE THREAD

**Issue by ZeroL04** [xylo/intellij-postfix-templates#224]

**Comment by xylo** [comment]
I just tested the JavaScript and TypeScript support in WebStorm and everything worked.  What exactly does not work for you in WebStorm?

**Comment by ZeroL04** [comment]
Maybe my webstorm version is too low, 2018.3.6, 2019.3.5, 2020.3.3 can't work properly

**Comment by xylo** [comment]
Yes, these versions are definitely too old.  Please try version 2022.2 oder 2022.3 EAP.

Unfortunately, I don't have the time to fix older versions than the current and the EAP release.

**Comment by xylo** [comment]
I will close the issue for now.  If the problem still exists in WebStorm 2022.2 or higher please let me know.

**Comment by ZeroL04** [comment]
OK, thanks
